src/components/WebexActivityStream/README.md

```diff
@@ -19,13 +19,19 @@ To see all the different possible states of the Webex Activity Stream component,
 1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
 
    ```js
-   const jsonAdapter = new RoomsJSONAdapter(rooms);
+   const WebexJSONAdapter = {
```

Comment:
I meant to say, this should be a `WebexJSONAdapter`, as in the object; not that you change the name of the variable.
```suggestion
   const jsonAdapter = new WebexJSONAdapter(jsonData);
```